web/js/roomController.js

```diff
@@ -955,7 +968,8 @@ PhoneNumberController, ResizeSensor, maxUsersPerRoom */
         .catch(function (error) {
           debug.error('Error Connecting to room. ' + error.message);
         });
-    });
+    })
+  .then(loadHotjar());
```

Comment:
Please change this to `.then(loadHotjar)`. (The parens for the function name are not needed.)

Reply:
I notice that if you click the Feedback button early enough (right away, before the precall controls are displayed) in the room view on Rob's Heroku instance, it works. But after a second or so, it does not. I thought maybe this was a z-index CSS issue ... but I do not see this issue when running locally.